content/attacks/posts/2021-05-19-PancakeBunny.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2021-05-19
+target-entities: PancakeBunny
+entity-types: DeFi
+attack-types: Flash Loan Attack
+title: "PancakeBunny suffers a flash loan attack"
+---
+
+## Summary
+
+On May 19, 2021 PancakeBunny, a yield farming aggregator built on Binance Smart Chain decentralized finance (DeFi) protocol, suffered a flash loan attack. According to [the SlowMist report they shared after investigating the incident](https://slowmist.medium.com/slowmist-pancakebunny-hack-analysis-4a708e284693) 
+> The attacker first borrows a huge amount of WBNB tokens from the multiple liquidity pools of PancakeSwap and borrows a huge amount of WBNB tokens from the Fortube project.
+>
+These assets were used to manipulate the price of BUNNY/BNB and USDT/BNB.
```

Comment:
> These assets were used to manipulate the price of BUNNY/BNB and USDT/BNB.

Rewrite as `These assets were used to manipulate the price of BUNNY`

Reply:
It's just i write about the double price manipulation in the next sentences, may i rewrite as "These assets were used to manipulate the price of BUNNY/BNB and USDT/BNB which ultimately caused BUNNY's price to collapse"?
But maybe this is when I can no longer hide my poor understanding:)

Reply:
No, since BUNNY/BNB is a pool, and pools have no prices. The token (BUNNY) has a price and was manipulated

Reply:
I see, I've rewritten everything, is it okay?